### PR TITLE
added language translation component

### PR DESCRIPTION
**Pull request by halyna-k**

### CODE REVIEW

File: src/components/Language/Language.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import HeaderButton from "../HeaderButton/HeaderButton"
+
+const Language = () => {
+  const intl = useIntl();
+  const iconFlag = intl.messages.flag
+
+  const handleChange = () => {
+    return (
+      console.log(intl)
```
Comment (dimalian) [inline src/components/Language/Language.js:11]: Please, remove this console.log

File: src/components/Authentication/Authentication.js [hunk 2]
```diff
@@ -12,23 +14,18 @@ const Authentication = () => {
   }
   console.log(user)
```
Comment (dimalian) [inline src/components/Authentication/Authentication.js:15]: please, don't leave test logs

File: src/components/Language/Language.js (hunk 1 above)
Comment (dimalian) [inline src/components/Language/Language.js:11]: same here

File: src/components/Authentication/Authentication.js (hunk 2 above)
Comment (halyna-k) [reply src/components/Authentication/Authentication.js:15]: Ok. I won't do it next time.

**Pull request merged**